ebu_tt_live/ui/user_input_producer/uip.js

```diff
@@ -423,8 +385,10 @@ $(document).ready(function() {
     $("#scheduled-send-clock-selector").change(handleScheduledSendSetupDependingOptions);
 
     $("#synchronize-media-clock-button").click(function() {
-        scheduled_send_media_clock_offset = Date.now();
-        stopResetRunningClock();
+        scheduled_send_media_clock_offset = moment();
+        //stopResetRunningClock();
+        $('#running-clock').hide();
+        $('#running-clock-media').show();
         interval_running_clock = setInterval(updateRunningClockMedia, 500);
```

Comment:
Why do we need both a `setInterval()` _and_ a `setTimeout()` (lines 116 and 123)? Seems like we should need one or the other but not both... Also, it looks as though every time the `synchronize-media-clock-button` is clicked another interval is created regardless of whether or not one already existed. I think this is leaky and probably consumes resources unnecessarily, especially because the current interval is not cleared (now that line 389 is commented out). Or possibly I have not understood...